### PR TITLE
Updated topics with _Overrides explanation

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/creating-artboard-adobexd.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ Open Adobe XD and select an iPhone 12 Pro Max. This will give you a new file wit
 
 ### 3. Add and customize components.
 
+> [!WARNING]
+> Symbols and components designated as `_Overrides` must not be inserted on artboards as they provide components states and do not represent to complete a component by itself.
```
Comment (StefanIvanov) [inline en/creating-artboard-adobexd.md:72]:
```suggestion
> Symbols and components designated as `_Overrides` must not be inserted on artboards as they provide component states and do not represent a complete component by itself.
```

File: en/creating-artboard-sketch.md [hunk 2]
```diff
@@ -75,6 +75,9 @@ You may have noticed that in the image above, the library also appears to be pre
 
 ### 3. Add and customize components.
 
+> [!WARNING]
+> Symbols and components designated as `_Overrides` must not be inserted on artboards as they provide components states and do not represent to complete a component by itself.
```
Comment (StefanIvanov) [inline en/creating-artboard-sketch.md:79]:
```suggestion
> Symbols and components designated as `_Overrides` must not be inserted on artboards as they provide component states and do not represent a complete component by itself.
```

File: en/components/components-overview.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
+The **Indigo.Design System** empowers your design workflow with 60+ UI Components that also map to Ignite UI for Angular controls. Each component is optimized for responsive web development and provides various templating capabilities through smart layout and resizing rules set in the library. Furthermore, we have added light/dark modes and various interaction states to choose from when designing dynamic experiences. All this streamlines application design and development allowing any user interface created with the library in Sketch to be built in Angular with Ignite UI.
+
+> [!WARNING]
+> Symbols and components designated as `_Overrides` in Sketch and Adobe XD libraries must not be inserted on artboards as they provide components states and do not represent to complete a component by itself.
```
Comment (StefanIvanov) [inline en/components/components-overview.md:12]:
```suggestion
> Symbols and components designated as `_Overrides` in Sketch and Adobe XD libraries must not be inserted on artboards as they provide component states and do not represent a complete component by itself.
```

File: en/components/components-overview.md [hunk 4]
```diff
@@ -15,7 +18,8 @@ The Sketch library has a `🧩 Components` page containing all Components and sy
...
-> Unlinking any component instance from the library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Detaching a component from the Sketch library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Ungrouping a component or pattern from the Adobe XD library has even worse implications because it turns all nested components into groups and breaks the code generation service for the component as it cannot trace correctly its structure. You can make any state, layout and styling adjustment in Adobe XD without ungrouping a component so we kindly advise you to keep the entirety of the components on your artboards.
```
Comment (StefanIvanov) [inline en/components/components-overview.md:22]: @YoannaIvanova could you add this second part as a warning in the patterns overview as well?

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/components-overview.md (hunk 4 above)
Comment (YoannaIvanova) [reply en/components/components-overview.md:22]: Added because of duplication in the PR https://github.com/IgniteUI/indigo-design-docfx/pull/528

**Review by StefanIvanov — APPROVED** [review]

**Pull request merged**